models/asset.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+type Asset struct {
+	ID        uint `gorm:"primary_key;"`
+	UserId    int  `gorm:"not null;"`
```

Comment:
Consistency with data is really important, the data values stored in the database must satisfy certain constraints. For example, the `user` listed for each `asset` must be one that actually exists. Do you think `assets` satisfies this? Why?

Reply:
I should have created a foreign key to `users` in order to ensure data integrity. Just a note though, for sure the FK is needed but the validation if a user exists or not most probably will take place on a middleware.